docs/src/developing/clients/jsonrpc-api.md

```diff
@@ -2813,7 +2831,7 @@ Before submitting, the following preflight checks are performed:
 
 #### Results:
 
-- `<string>` - Transaction Signature, as base-58 encoded string
+- `<string>` - First Transaction Signature embedded in the transaction, as base-58 encoded string (to be used for identification of transaction)
```

Comment:
Maybe it would be better to define something like "transaction ID" in https://docs.solana.com/terminology and link to it from here (and other places)?

Reply:
done!